errors/Makefile.am

```
@@ -24,6 +24,12 @@ EXTRA_DIST = \
 	template.am \
 	aliases alias-link.sh alias-upgrade errorpage.css TRANSLATORS COPYRIGHT
 
+if ENABLE_TRANSLATION
+ALIAS_LINKER=$(SHELL) $(srcdir)/alias-link.sh "$(LN)" "$(RM)" "$(DESTDIR)$(DEFAULT_ERROR_DIR)"
+else !ENABLE_TRANSLATION
+ALIAS_LINKER=true
```

The shell keyword ```true``` is not always an executable. autoconf provides a $(TRUE) replacement we should use instead to handle the edge cases properly.
```suggestion
ALIAS_LINKER=$(TRUE)
```